packages/shared/src/components/comments/CommentBox.tsx

```diff
@@ -76,11 +79,32 @@ function CommentBox({
         <div className="flex flex-col ml-3 typo-callout">
           <FlexRow>
             <CommentAuthor
-              postAuthorId={postAuthorId}
               author={comment.author}
               appendTooltipTo={appendTooltipTo}
+              badges={[
+                <SquadMemberRoleBadge
+                  key="squadMemberRole"
+                  source={post.source}
+                  author={comment.author}
+                />,
+                comment.author.id === postAuthorId && (
+                  <UserBadge
+                    key="author"
+                    className="text-theme-status-help"
+                    content="Author"
+                    icon={<FeatherIcon secondary className="mx-0.5" />}
+                  />
+                ),
+                comment.author.id === postScoutId && (
+                  <UserBadge
+                    key="scout"
+                    className="text-theme-color-bun"
+                    content="Scout"
+                    icon={<ScoutIcon className="mr-0.5" />}
```

Comment:
Why do these icons have different classes should they not be similar? (seeing they conditional render)